collector/utils.py

```diff
@@ -13,10 +14,22 @@
 # pylama:ignore=E1101
 requests.packages.urllib3.disable_warnings()
 
+if not (REDIS_ENV and REDIS_PASSWORD):
+    LOGGER.error('Environment not set properly for Redis')
+    sys.exit(1)
```

Comment:
@AparnaKarve I don't think this exit will work. This will just loop the container execution restarting, while it wont cause the container failed. I think we already have a fix for this and you've already been using it in this PR https://github.com/ManageIQ/aiops-publisher/pull/15 (commit https://github.com/ManageIQ/aiops-publisher/commit/8d7c20997ee78c60a9c93b3f6243a334a0a5c58b).